tests/common.go

```diff
@@ -4551,6 +4551,79 @@ func HaIncreaseRebootSourceNode(event *EventRecord, ctx *scheduler.Context, v *v
 		})
 }
 
+// HaIncreaseRebootPX repl increase and reboot px on node
+func HaIncreaseRebootPXOnNode(event *EventRecord, ctx *scheduler.Context, v *volume.Volume, storageNodeMap map[string]node.Node) {
```

Comment:
change the eixisting function taking testType reboot or restart.
if reboot. reboot the node else restart px